nixos/infrastructure/vagrant.nix

```diff
@@ -39,7 +39,7 @@ in {
     # These groups are created through the ENC normally which
     # doesn't exist in vagrant or at least here, yet.
     users.groups = {
-      login = {};
+      login = { members = ["vagrant"]; };
```

Comment:
Hmm. This seems to be a duplicate. Line 51 already assigns login as an extra group to the vagrant user.

Reply:
Actually I meant to the opposite of what happened now: why are you extracting those here? Is the code on line 51 buggy somehow?

Reply:
It was. The logic behind generating the htpasswd-file actually did not respect the settings done with `users.users.vagrant.extraGroups` -- in an update I moved login and service out of the extraGroups, kept docker.

Reply:
Ugh. "interesting". Alright then. Actually that means we should be writing a test for that htpasswd generation code and maybe fix that bug ...